Sources/MessagingInApp/Views/InAppMessageView.swift

```diff
@@ -47,6 +47,13 @@ public class InAppMessageView: UIView {
     // Delegate to handle custom action button tap.
     public weak var onActionDelegate: InAppMessageViewActionDelegate?
 
+    // When a fetch request is performed, it's an async operation to have the inline View notified about this fetch and the inline View processing the fetch.
+    // There is currently no easy way to know when the inline View has finished processing the fetch.
+    // This listener is a hack for our automated tests to know when the inline View has finished processing the fetch.
+    //
+    // See linear ticket MBL-427 to learn more about this limitation in our tests.
```

Comment:
As mentioned in comments, this listener is added because of async operations being performed and our tests needing to wait for them to complete. Instead of something like `sleep`, we have a listener the tests can hook into. 

This solution is not ideal but it's a simple solution to unblock integration tests for inline Views. 

See ticket https://linear.app/customerio/issue/MBL-427/ for ideas on how we can improve this solution.